sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizedForm.java

```diff
@@ -90,4 +90,23 @@ public FormPageRange getPageRange() {
     public List<FormPage> getPages() {
         return this.pages;
     }
+
+    /**
+     * Get the confidence of the form type identified by the model.
+     *
+     * @return the formTypeConfidence value.
+     */
+    public Float getFormTypeConfidence() {
```

Comment:
Does Java has nullable types? we set this type as `Float?` in .NET. 
I also think the docstring could be something similar to: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizedForm.cs#L82

to help the user

Reply:
`Float` is the nullable type, the other is `float` which would have been non-nullable. And here since the service may not send this back kept it as nullable.
I will update the javadocs to say something similar